<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetDollarUI</class>
 <widget class="QWidget" name="GetDollarUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetDollar</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>81</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>28</number>
    </property>
    <item>
     <widget class="QCheckBox" name="usd_checkbox">
      <property name="text">
       <string>USD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="eur_checkbox">
      <property name="text">
       <string>EUR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="jpy_checkbox">
      <property name="text">
       <string>JPY</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>160</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="usd_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="eur_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="jpy_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
